app/assets/stylesheets/darkswarm/_shop-navigation.css.scss

```diff
@@ -69,23 +71,26 @@ ordercycle {
 
     select,
     p {
-      width: inherit;
       display: inline-block;
       color: $white;
       background-color: transparent;
       border: 0;
-      margin-bottom: 0;
+      border-radius: 0 0.25em 0.25em 0;
```

Comment:
you can reuse the var `$radius-small` here defined in https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/assets/stylesheets/darkswarm/variables.css.scss#L36

Reply:
I rebased this PR to resolve conflicts and this line was exactly where the conflict was and guess what the line in master looked like? same but with $radius-small :-)